docker-compose.yml

```diff
@@ -5,13 +5,19 @@ networks:
 services:
   pythonapp:
     image: todoapp:2.0.0
-    build:
-      context: .
-      dockerfile: Dockerfile
+    build: .
+      #context: .
+      #dockerfile: Dockerfile
```

Comment:
Why you've changed build context ?

Reply:
this part has been commented on a QA session on Thursday with Nazar